netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -538,6 +545,59 @@ public void maxHeaderListSizeShouldBeEnforcedOnClient() throws Exception {
     }
   }
 
+  @Test
+  public void huffmanCodingShouldNotBePerformed() throws Exception {
+    startServer();
+
+    Callable<NettyClientTransport> nonHuffmanTransport =  () -> newTransport(newNegotiator(),
+        DEFAULT_MAX_MESSAGE_SIZE, GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE, null, true,
+        TimeUnit.SECONDS.toNanos(10L), TimeUnit.SECONDS.toNanos(1L),
+        new ReflectiveChannelFactory<>(NioSocketChannel.class), group);
+
+    Metadata aHeaders = new Metadata();
+    aHeaders.put(Metadata.Key.of("test", Metadata.ASCII_STRING_MARSHALLER),
+        LONG_STRING_OF_A);
+
+    Metadata zHeaders = new Metadata();
+    zHeaders.put(Metadata.Key.of("test", Metadata.ASCII_STRING_MARSHALLER),
+        LONG_STRING_OF_Z);
+
+    Metadata warmupHeaders = new Metadata();
+    warmupHeaders.put(Metadata.Key.of("test", Metadata.ASCII_STRING_MARSHALLER), "unused");
+
+    //warm up hpack
```

Comment:
What does this do? It's on a different connection so is throw away, and nothing seems timing sensitive. So I'm not sure what part of hpack is being warmed.

Reply:
This was an artifact of my attempts to stabilize the tests. I was trying different combinations of new/re-used connections, so this would handle the overhead of the initial settings frames.

It's now removed 🙂